feasibility-dsf-process/src/main/java/de/medizininformatik_initiative/feasibility_dsf_process/service/ObfuscateEvaluationResult.java

```diff
@@ -35,11 +41,32 @@ protected void doExecute(DelegateExecution execution, Variables variables) {
     }
 
     private MeasureReport obfuscateFeasibilityCount(MeasureReport measureReport) {
-        var obfuscatedFeasibilityCount = obfuscator.obfuscate(measureReport.getGroupFirstRep().
-                getPopulationFirstRep().getCount());
+        var obfuscatedFeasibilityCount = obfuscator.obfuscate(extractInitialPopulation(measureReport).getCount());
+
+        var obfuscatedMeasureReport = new MeasureReport()
+                .setMeasure(measureReport.getMeasure())
+                .setType(measureReport.getType())
+                .setStatus(measureReport.getStatus())
+                .setDate(measureReport.getDate())
```

Comment:
I'm not sure whether we are allowed to expose the date. Please ask Phillip / Julian.

Reply:
@juliangruendner & @kleinertp  please comment

Reply:
@EmteZogaf : remove the date